lib/control/control-commands.c

```diff
@@ -58,6 +58,25 @@ control_register_command(const gchar *command_name, const gchar *description, Co
   command_list = g_list_append(command_list, new_command);
 }
 
+void
+control_replace_command(const gchar *command_name, const gchar *description, CommandFunction function,
+                        gpointer user_data)
+{
+  GList *command_it =  g_list_find_custom(command_list, command_name,
+                                          (GCompareFunc)control_command_start_with_command);
+  if (!command_it)
+    {
+      msg_error("Failed to replace control command",
```

Comment:
hmm... I think we could register the command when we want to replace a non-existent (and print a warning msg): I'll change it.